### PR TITLE
Keybinds are now added to CONFIG file

### PR DESCRIPTION
**Pull request by TheRiddler4K**

### CODE REVIEW

File: my_game.toml [hunk 1]
```diff
@@ -17,6 +17,10 @@ PLAYER_SPEED_LIMIT = 5  # px/update
 PLAYER_INVINCIBILITY_SECONDS = 5  # secs
 PLAYER_FIRE_RATE = 0.2  # secs
 PLAYER_SHOT_RANGE = 400  # px
+PLAYER_THRUST_KEY = 65362
+PLAYER_FIRE_KEY = 32
+PLAYER_TURN_RIGHT_KEY = 65363
+PLAYER_TURN_LEFT_KEY = 65361
```
Comment (tobiasbp) [inline my_game.toml:24]: Please add a link to the key numbers here: https://api.arcade.academy/en/latest/arcade.key.html#key
Also add comments so a human can understand the current settings:
_PLAYER_FIRE_KEY = 32 # Space_

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.toml (hunk 1 above)
Comment (TheRiddler4K) [reply my_game.toml:24]: Ok

Comment (TheRiddler4K) [reply my_game.toml:24]: Im not sure i understand

Comment (tobiasbp) [reply my_game.toml:24]: Add comments in the config file. That way, other humans can understand it better.

Comment (TheRiddler4K) [reply my_game.toml:24]: Ok that makes sense

**Comment by TheRiddler4K** [comment]
I believe you can also do this with the arcade.color package in the form of three byte numbers (RGB system) and four byte numbers (RGB systems and transparency) in tuples

**Review by tobiasbp — APPROVED** [review]
Good stuff. Thanks.

**Pull request merged**